### PR TITLE
Add a parser for TMCF

### PR DESCRIPTION
**Pull request by shifucun**
This only supports resolved TMCF that does not have entity to entity reference.

### CODE REVIEW

File: internal/parser/tmcf.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+			}
+			result[table].ColumnInfo[parts[1]] = &Column{Node: node, Property: head}
+		} else if strings.HasPrefix(body, base.DCS) {
+			// Schema mapping
+			schema := strings.TrimPrefix(body, base.DCS)
+			if _, ok := result[table].NodeSchema[parts[1]]; !ok {
+				result[table].NodeSchema[node] = map[string]string{}
+			}
+			result[table].NodeSchema[node][head] = schema
+		}
```
Comment (pradh) [inline internal/parser/tmcf.go:92]: This can be any of: namespace prefix (dcs:, dcid:, schema:), string, number

File: internal/parser/tmcf.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+				return nil, status.Errorf(codes.Internal, "Invalid input for Column:\n%s", line)
+			}
+			if table == "" || node == "" || table != parts[0] {
+				return nil, status.Errorf(codes.Internal, "Invalid input for Column:\n%s", line)
+			}
+			result[table].ColumnInfo[parts[1]] = &Column{Node: node, Property: head}
+		} else if strings.HasPrefix(body, base.DCS) {
+			// Schema mapping
+			schema := strings.TrimPrefix(body, base.DCS)
+			if _, ok := result[table].NodeSchema[parts[1]]; !ok {
```
Comment (pradh) [inline internal/parser/tmcf.go:88]: can check that table and node are not empty here.

**Review by pradh — APPROVED** [review]

File: internal/parser/tmcf.go (hunk 2 above)
Comment (shifucun) [reply internal/parser/tmcf.go]: Done

File: internal/parser/tmcf.go (hunk 1 above)
Comment (shifucun) [reply internal/parser/tmcf.go:103]: I see. changed to removing these prefix

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**